sdk/src/main/java/com/qonversion/android/sdk/internal/QProductCenterManager.kt

```diff
@@ -673,6 +723,9 @@ internal class QProductCenterManager internal constructor(
         return object : QonversionLaunchCallback {
             override fun onSuccess(launchResult: QLaunchResult) {
                 handledPurchasesCache.saveHandledPurchases(trackingPurchases)
+                trackingPurchases.forEach {
+                    removePurchaseOptions(it.productId)
+                }
```

Comment:
Here on success, we clear tracked purchases options, while onError (in opposite to purchase) we don't. Shouldn't we clear options for the tracking purchases on init error too?

Reply:
No, because it's init function, and we don't use any callbacks here, so we will retry to send purchases next time, and we should keep options for the next try.